app/services/utils/raster.py

```diff
@@ -0,0 +1,51 @@
+from rio_tiler.io import Reader
+from rasterio import features
+from geopandas import GeoDataFrame
+from shapely import geometry
+
+
+# TODO: become generic in order to be able to reuse
+def crop_raster(raster_path, polygon):
+    with Reader(raster_path) as image:
+        img = image.feature(polygon)
+
+    return img.render(
+        add_mask=True,
+        colormap={
+            0: (255, 0, 0, 255),
+            1: (128, 204, 102, 255),
+            2: (232, 214, 107, 255),
+        },
+    )
+
+
+def get_raster_values(raster_path, polygon):
+    crs = "EPSG:9377"
+    categories = {"Perdida": 0, "Persistencia": 1, "No bosque": 2}
+
+    with Reader(raster_path) as cog:
+        data = cog.feature(polygon, dst_crs=crs)
+        transform = data.transform
+        mask = data.mask != 0
```

Comment:
what's the result of this operation? I guess is not a boolean

Reply:
This operation removes the zeros present in the mask, converting them to False to filter the valid areas of the raster. [Creating our boolean mask](https://medium.com/@twelsh37/python-masking-data-before-plotting-6fd8de196aca)